src/main/flight/mixer.c

```diff
@@ -657,7 +664,7 @@ static void calculateThrottleAndCurrentMotorEndpoints(timeUs_t currentTimeUs)
 #endif
 
         currentThrottleInputRange = rcCommandThrottleRange;
-        motorRangeMin = motorOutputLow + motorRangeMinIncrease * (motorOutputHigh - motorOutputLow);
+        motorRangeMin = dynIdleMotorOutputLow + motorRangeMinIncrease * (motorOutputHigh - dynIdleMotorOutputLow);
```

Comment:
Naming of this seems to be a bit confusing - this is applied even when dynamic idle is not enabled - maybe rather `appliedMotorOutputLow`?

Reply:
Ok, updated.